homework_11/homework_11.py

```diff
@@ -0,0 +1,82 @@
+"""
+Написати власний декоратор, задачею якого має бути друк назви функції i часу, коли вона була викликана. Декоратор має працювати для різних функцій однаково.
+
+Написати кастомний Exception клас, MyCustomException, який має повідомляти "Custom exception is occured".
+
+Написати власний менеджер контексту, задачею якого буде друкувати "=========="  10 знаків дорівнює перед виконанням коду та після виконання коду, таким чином виділяючи блок коду символами дорівнює.
+
+Y випадку виникнення будь-якої помилки вона має бути надрукована текстом, проте програма не має завершити своєї роботи.
+
+Написати конструкцію try ... except ... else ... finally, яка буде робити точно те ж, що i менеджер контексту із попереднього завдання.
+"""
+
+#(1)
+
+import time
+
+def decorator(func):
+    def wrapper():
+        print("current time is:", time.strftime('%H'':''%M'))
+        func()
```

Comment:
Add here `return` and pass here *args and kwargs to be able to work with functions that return some value